.github/workflows/update-template.yml

```diff
@@ -79,22 +79,37 @@ jobs:
         id: run-update-template
         run: |
           npm v ${{ needs.parse-issue-form-body.outputs.npm-package }} dist.tarball | xargs curl | tar -xz
-          node src/update-template.js $GITHUB_WORKSPACE/package ${{ needs.parse-issue-form-body.outputs.github-link }} ${{ needs.parse-issue-form-body.outputs.npm-package }}
+          node src/update-template.js $GITHUB_WORKSPACE/package ${{ needs.parse-issue-form-body.outputs.github-link }} ${{ needs.parse-issue-form-body.outputs.npm-package }} ''
```

Comment:
@AmyJZhao , should a status be set to TEMPLATE_STATUS_IN_VERIFICATION for 3rd-party template updates? as we support the latest template versions only, we should be sure that the latest template version works

Reply:
I changed it so that it'll set TEMPLATE_STATUS_IN_VERIFICATION for 3rd party templates